octoprint_firmwareupdate/__init__.py

```diff
@@ -265,6 +265,10 @@ def _update_firmware(self, target):
         if not self.isUpdating:
             self._logger.info("Skipped initiation. Aborting...")
         else:
+            if target is "local":
+                # Using something other than GitHub, delete version file
+                self._delete_version_file()
+
```

Comment:
Is this to ensure that it pulls the latest version file from GitHub after a custom fw file was loaded?